Modules/Scripted/VPAWVisualize/vpawvisualizelib/isosurfaces.py

```diff
@@ -0,0 +1,81 @@
+import vtk
+import slicer
+
+def isosurfaces_from_volume(vol_node, thresholds, decimate_target_reduction = 0.25, progress_callback = None):
+    """ Compute a model node consisting of isosurfaces from the given volume node.
+    Uses vtkFlyingEdges3D to generate isosurface mesh.
+
+    Args:
+        vol_node: a vtkMRMLScalarVolumeNode
+        thresholds: a sequence of floats; values at which to threshold the scalar volume. each value
+            should result in one isosurface
+        decimate_target_reduction: by how much to decimate after doing vtkFlyingEdges3D
+        progress_callback: Optionally, a function that takes a progress_percentage float value.
+            If this is provided then progress_callback(progress_percentage) will be called
+            while the computation is being done.
+    Return: a vtkMRMLModelNode
+    """
+
+    if progress_callback is None:
+        progress_callback = lambda progress_percentage : None
```

Comment:
This is probably okay as is, but `flake8` doesn't like that a lambda is being assigned a name.  It prefers that we use `def` to assign names to functions.  Maybe this isn't the situation that `flake8` means to catch, but I thought to mention it anyway.